components/Datasets/DescribeDataset.vue

```diff
@@ -346,8 +346,8 @@
           >
             <template v-for="(contact_point, index) in form.contact_points">
               <ContactPointSelect
-                v-if="contact_point.role == 'contact'"
-                :key="'id' in contact_point ? contact_point.id : index"
+                v-if="!contact_point || contact_point.role == 'contact'"
```

Comment:
Will need to be updated when merged with https://github.com/datagouv/front-end/pull/317